api/src/seeders/3-user.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+const config = require('../config/server.json');
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('users', [
+      {
+        name: 'The Admin',
+        email: 'admin@crate.com',
+        password: bcrypt.hashSync('123456', config.saltRounds),
+        thirty_days_from: new Date(),
+        ballotTitle: "asdf",
+        ballotDescription: "really cool stuff",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        meetUpAddress: "123 3rd street"
+      },
+      {
+        name: 'The User',
+        email: 'user@crate.com',
+        password: bcrypt.hashSync('123456', config.saltRounds),
+        thirty_days_from: new Date(),
+        ballotTitle: "asdf",
+        ballotDescription: "really cool stuff",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        meetUpAddress: "123 3rd street"
+      }
+    ])
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.bulkDelete('users', null, {});
+  }
+}
```

Comment:
Seeds to ensure the DB is setup correctly and data can be mutated/queried

Reply:
Ok, so were going to have users logging in?

Reply:
Yeah, just for the one's who want to go through the initiative process so we can save their initiative draft/meetupdate/ etc but not through GoogleOauth or anything just something internal